src/utils/Api.ts

```diff
@@ -17,7 +17,7 @@ export default class Api {
   private chainId: string
 
   constructor(abis: HexAbi[], publicKeys: string[], chainId: string) {
-    this.rpc = new JsonRpc(null) // provide null chainURL to ensure vault isn't reaching out to chain for info
+    this.rpc = new JsonRpc('') // provide null chainURL to ensure vault isn't reaching out to chain for info
```

Comment:
Why must this be empty string?  What was changed that required this?  Was that change really necessary?
Also, *if* this is necessary, let's move this into the function as a default arg value to make the function robust against bad input and not burden clients with having to know to add this arg to their calls.

Reply:
I've made PRs on eosjs to resolve the issue on eosjs side (two options to either allow or error)
https://github.com/EOSIO/eosjs/pull/631
https://github.com/EOSIO/eosjs/pull/632

As for this package, I believe changing it to an empty string is the better option since the JsonRpc constructor is expecting a string for endpoint with TS.  Using null instead is not good practice and is circumventing type checking.  I'm updating to change "provide null chainURL" to "provide empty chainURL" though.

Reply:
https://github.com/EOSIO/eosio-reference-chrome-extension-authenticator-app/pull/37/files/ead38bb2420c82514a8d719d5d2bc424a04c564c..d8455f0e00255a5be1781e1643d298daf1572df3#diff-db1f72b58945ed2936a1519c7c18f11c